package.json

```diff
@@ -112,7 +112,7 @@
     "topicSeparator": " ",
     "topics": {
       "attest": {
-        "description": "[Attest]       Attest to 'Not Reviewed' controls in order to account for all of your controls"
+        "description": "[Attest]       Attesting to 'Not Reviewed' control requirements (that can’t be tested automatically by security tools and hence require manual review), helping to account for all requirements"
```

Comment:
Do we want to intentionally break our pattern of using the exact verb?  (i.e. going from attest to attesting).  I think the sentence works fine even with 'Attest'

Reply:
```suggestion
        "description": "[Attest]       Attest to 'Not Reviewed' control requirements (that can’t be tested automatically by security tools and hence require manual review), helping to account for all requirements"
```

Reply:
done